### ISSUE TITLE
[Timeline] button.timeline-step-title-action wrong apparence

### ISSUE THREAD

**Issue by BertrandPodevin** [LuccaSA/lucca-front#1623]
`timeline-step-title-action `wich are used to navigated between the steps of a `timeline` can be `button`.

The apparence is currently broken when doing so. It's only working with `a`

![image](https://user-images.githubusercontent.com/65592565/164676011-c3ea5c9a-eee4-421e-883d-3daa0e05b033.png)

**Comment by BertrandPodevin** [comment]
Can be fixed with 
```
.timeline-step-title-action {
  @include buttonReset;
  outline: none;
  &[disabled] {
      text-decoration: none;
      cursor: default;
  }
}
```

**Comment by jeremie-lucca** [comment]
Fixed (8.1.5) https://github.com/LuccaSA/lucca-front/pull/1631
